# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/mergeant/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

--- mergeant-0.12.1/src/interface_cb.c.orig	2004-05-10 15:59:39.247566552 +0200
+++ mergeant-0.12.1/src/interface_cb.c	2004-05-10 16:00:02.963961112 +0200
@@ -2210,5 +2210,6 @@
 						   gda_connection_get_errors (cnc));
 		break;
 	default :
+		;
 	}
 }
